### DISCUSSION TITLE
Why Zero Balance?

### DISCUSSION THREAD

**Discussion by kamisiano11** [iotaledger/legacy-migration-tool#132] [Help & Support]
Hello.  The legacy migration tool accepted my 81 character seed, but shows my balance to be zero.  I've checked and rechecked my seed and have logged on numerous times, always with a zero balance.  This is not true as I have a balance of 156.3+Mi on Trinity.  Thoughts?

**Comment by begonaalvarezd** [comment]
If you are not finding your balance after plenty searches, my guess is that the 81 chars you are using is incorrect, you already migrated, or you moved the funds somewhere in the past in trinity
You can use the explorer to check your legacy trinity address if you know it https://explorer.iota.org/legacy-mainnet, if you have transactions after april 2021 or so it means you already migrated

**Comment by kamisiano11** [comment]
Hello,I found the same balance on Explorer as in my Trinity Wallet (see below).  I can’t see how I copied my 81 character seed down incorrectly, unless what I thought were the letter O were zeros.  Since Trinity Wallet gives me an option to send, could I make a new Firefly account and send my balance there or is that not sufficient for migration?   Thoughts?KevinOn Feb 26, 2024, at 04:00, Begoña Álvarez de la Cruz ***@***.***> wrote:﻿
If you are not finding your balance after plenty searches, my guess is that the 81 chars you are using is incorrect, you already migrated, or you moved the funds somewhere in the past in trinity
You can use the explorer to check your legacy trinity address if you know it https://explorer.iota.org/legacy-mainnet, if you have transactions after april 2021 or so it means you already migrated

**Comment by kamisiano11** [comment]
Hello,I have access to my 81 character seed when I log in to Trinity wallet.  Since all characters must be either alpha A-Z or the number 9, I am confident that I am typing precisely what Trinity has displayed under my “Account Management, View Seed” into the IOTA Legacy Migration Tool.  However, the IOTA Legacy Migration Tool continues to say my balance is zero when it is not.Do you work directly for IOTA?KevinOn Feb 28, 2024, at 17:40, Kevin Misiano ***@***.***> wrote:﻿Hello,I found the same balance on Explorer as in my Trinity Wallet (see below).  I can’t see how I copied my 81 character seed down incorrectly, unless what I thought were the letter O were zeros.  Since Trinity Wallet gives me an option to send, could I make a new Firefly account and send my balance there or is that not sufficient for migration?   Thoughts?Kevin<image1.png>On Feb 26, 2024, at 04:00, Begoña Álvarez de la Cruz ***@***.***> wrote:﻿
If you are not finding your balance after plenty searches, my guess is that the 81 chars you are using is incorrect, you already migrated, or you moved the funds somewhere in the past in trinity
You can use the explorer to check your legacy trinity address if you know it https://explorer.iota.org/legacy-mainnet, if you have transactions after april 2021 or so it means you already migrated

**Reply by begonaalvarezd** [reply]
Are you sure you didnt migrate already? The migrations started in april 2021 so if in the explorer shows for you a tx at that time or after, you probably already migrated, that is the only explanation I can find for your balance not showing up, or that your seed is not correct.

> Do you work directly for IOTA

The repository is under the official IOTA Foundation github repository, so yes, we are an official tool

**Comment by M-Nassir** [comment]
I have a similar issue. After putting in my 81 character seed, with the correct checksum, the balance shows as 0. However, on the tango explorer I can input my receive address and see the correct balance. The funds are inaccessible. Makes me wonder about crypto in general, seems like you can come back after a few years and find your funds are inaccessible unlike real world hard assets!
